tox.ini

```diff
@@ -89,7 +89,7 @@ commands =
     pip install -e .
     # TODO: Workaround for https://github.com/ethereum/py-solc/issues/64
     pip install git+https://github.com/Jonasmpi/py-solc.git
-    pip install git+https://github.com/gridsingularity/d3a-interface.git
+    pip install git+https://github.com/gridsingularity/d3a-interface.git@feature/D3ASIM-973
```

Comment:
Please remove this before you merge to master

Reply:
Will do. This is just to validate that travis tests pass.